internal/provider/model_fmc_hosts.go

```diff
@@ -0,0 +1,258 @@
+// Copyright © 2023 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Mozilla Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://mozilla.org/MPL/2.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: MPL-2.0
+
+package provider
+
+// Section below is generated&owned by "gen/generator.go". //template:begin imports
+import (
+	"context"
+	"fmt"
+	"maps"
+
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/tidwall/gjson"
+	"github.com/tidwall/sjson"
+)
+
+// End of section. //template:end imports
+
+// Section below is generated&owned by "gen/generator.go". //template:begin types
+
+type Hosts struct {
+	Id     types.String          `tfsdk:"id"`
+	Domain types.String          `tfsdk:"domain"`
+	Items  map[string]HostsItems `tfsdk:"items"`
+}
+
+type HostsItems struct {
+	Id          types.String `tfsdk:"id"`
+	Description types.String `tfsdk:"description"`
+	Overridable types.Bool   `tfsdk:"overridable"`
+	Ip          types.String `tfsdk:"ip"`
+	Type        types.String `tfsdk:"type"`
+}
+
+// End of section. //template:end types
+
+// Section below is generated&owned by "gen/generator.go". //template:begin getPath
+
+func (data Hosts) getPath() string {
+	return "/api/fmc_config/v1/domain/{DOMAIN_UUID}/object/hosts"
+}
+
+// End of section. //template:end getPath
+
+func (data Hosts) toBody(ctx context.Context, state Hosts) string {
```

Comment:
Why is this section no longer generated/updated?

Reply:
Default code would generate the following body:
```
{
"id": "...",
"items": [{host1_data},{host2_data}]
}
```
while I need just what's inside of items
```
[{host1_data},{host2_data}]
```

I guess, the easiest way to get back to the templated version is to extract items in resource, wherever the toBody() function in called. Is that a way to go?

Reply:
This is template generated now.